packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -593,6 +621,10 @@ export class TerminalFrontendContribution implements FrontendApplicationContribu
             keybinding: 'shift-pageDown',
             context: TerminalKeybindingContexts.terminalActive
         });
+        keybindings.registerKeybinding({
+            command: TerminalCommands.TOOGLE_TERMINAL.id,
+            keybinding: 'ctrl+shift+.',
```

Comment:
Where does <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>.</kbd> come from?

From what I can tell VS Code uses <kbd>ctrlcmd</kbd>+<kbd>`</kbd> and a different keybinding on macos:

https://github.com/microsoft/vscode/blob/705b635c5ef24aa6cde39f4209f725eca93b07b6/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts#L139-L146

Reply:
I used `ctrl`+`shift`+`.` for testing purposes. I don't really know how this decisions are made in the project and if we should use the same keybinding as in VSCode. Do you think I should define `ctrlcmd`+` (I don't know how to escape this character in Markdown) then?

Reply:
It looks like in VSCode, the keybinding is <kbd>Ctrl</kbd>+<kbd>`</kbd> on both Windows and MacOS, so that's probably the way to go.

https://github.com/microsoft/vscode/blob/7d9b1c37f8e5ae3772782ba3b09d827eb3fdd833/src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts#L138-L144

Reply:
Thank you! I have modified the keybinding to `Ctrl`+` as suggested.